Common/Tools/TrackTuner.h

```diff
@@ -538,11 +589,17 @@ struct TrackTuner : o2::framework::ConfigurableGroup {
     double dcaZPullMC = 1.0;
     double dcaZPullData = 1.0;
 
-    dcaXYResMC = evalGraph(ptMC, grDcaXYResVsPtPionMC.get());
-    dcaXYResData = evalGraph(ptMC, grDcaXYResVsPtPionData.get());
+    // get phibin
+    double phiMC = mcparticle.phi();
+    if (phiMC < 0.)
+      phiMC += o2::constants::math::TwoPI;                                    // 2 * std::numbers::pi;//
+    int phibin = phiMC / (o2::constants::math::TwoPI + 0.0000001) * nPhiBins; // 0.0000001 just a numerical protection
```

Comment:
Just for completeness: in Run2 we were using `TMath::Floor()`, we can use `std::floor` here, but maybe it's not necessary (doing some test locally the result does not change with or without it)


```suggestion
    int phiBin = phiMC / (o2::constants::math::TwoPI + 0.0000001) * nPhiBins; // 0.0000001 just a numerical protection
```

Reply:
ok for the phi bin, I do not think using floor is needed